easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -811,13 +812,13 @@ def test_step(self, return_output_ec=False):
                     raise EasyBuildError("Failed to create test install dir: %s", err)
 
                 # print Python search path (just debugging purposes)
-                run_cmd("%s -c 'import sys; print(sys.path)'" % self.python_cmd, verbose=False, trace=False)
+                run_shell_cmd("%s -c 'import sys; print(sys.path)'" % self.python_cmd, hidden=True)
 
                 abs_pylibdirs = [os.path.join(actual_installdir, pylibdir) for pylibdir in self.all_pylibdirs]
                 extrapath = "export PYTHONPATH=%s &&" % os.pathsep.join(abs_pylibdirs + ['$PYTHONPATH'])
 
                 cmd = self.compose_install_command(test_installdir, extrapath=extrapath)
-                run_cmd(cmd, log_all=True, simple=True, verbose=False)
+                run_shell_cmd(cmd, hidden=True)
```

Comment:
@bartoldeman Did you intentionally hide the installation command before running the tests in `PythonPackage.test_step`?

Just wondering, not opposing it, sort of makes sense, but it is different from what we had before (`verbose=False` in `run_cmd` is not equivalent with `hidden=True` in `run_shell_cmd`)